force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -69,7 +73,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The quotient of the division, or 0 if the denominator is zero.
      */
     public static Decimal division(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        if (b == 0) {
+            return 0;
+        }
+
+        return a / b; // Replace null with the variable you used to store the result
```

Comment:
Great work! Another thing you can try is using a ternary operator. 
`return (b == 0) ? 0 : a / b`

The expression to the left of the question mark gets evaluated and if its true, it returns the value immediately to the right of the question mark. If its false, it returns the value that is after the colon.